app/models/review.rb

```diff
@@ -11,7 +11,7 @@ class Review < ActiveRecord::Base
   validates :user, presence: true
   validates :beer, presence: true
 
-  def owner?(logged_in_user)
-    user == logged_in_user
+  def amendable?(logged_in_user)
+    logged_in_user == user || logged_in_user.admin?
```

Comment:
Ok you have this same method twice. Do you want to extract it out into a module somewhere? Not necessary right now, but put it on your list for refactoring. Once you get your tests running and working, you can refactor to your hearts' delights and run tests after each refactor.